snappy/security.go

```diff
@@ -319,6 +320,13 @@ func findWhitespacePrefix(t string, s string) string {
 	return subs[1]
 }
 
+// XXX: what to use really?
+func getSecurityProfile2(s *snap.Info, appName, baseDir string) string {
```

Comment:
This is obsolete after we switch to interfaces. We don't import anything from ``snappy/`` package in interfaces.